.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Vagrant
+.vagrant
```

Comment:
Why should we ignore vagrant files?

Reply:
It's a local ".vagrant" file that creates automatically, which means every computer who will run vagrant will create a local copy of it.